### DISCUSSION TITLE
time to get starstruck

### DISCUSSION THREAD

**Discussion by aarongeo1** [Schweinepriester/github-profile-achievements#425] [General]
star my repo and I will do the same to yours, lets get it guys together
Here is my repo of a AI that predicts handwritten test data of numbers
https://github.com/aarongeo1/NUM-CLASSIFY

**Comment by xLevitate** [comment]


**Reply by aarongeo1** [reply]
done thankyou

**Comment by drk1rd** [comment]
Done!
https://github.com/drk1rd/drk1rd
